openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1167,14 +1167,14 @@ The Credential Issuer MUST invalidate the `transaction_id` after the Credential
 The following is a non-normative example of a Deferred Credential Request:
 
 ```
+POST /credential HTTP/1.1 
```

Comment:
`/deferred_credential` will probably work better in this example.

Reply:
```suggestion
POST /deferred_credential HTTP/1.1 
```